fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -78,6 +81,33 @@ def cli(ctx: click.Context, config_path: str, local: bool) -> None:
     if not ctx.invoked_subcommand:
         click.echo(cli.get_help(ctx))
 
+    if (
+        ctx.obj["CONFIG"].user.analytics_opt_out is None
+        and ctx.invoked_subcommand != "init"
+    ):
+        ctx.obj["CONFIG"].user.analytics_opt_out = bool(
```

Comment:
does this defaulting to opting in need to be documented so users are aware?

Reply:
This does not default to opting in, it enforces a requirement that the user makes a decision to opt in or out.